### PR TITLE
Remade getBattleAttack(), fixed some formatting, commented

### PR DESCRIPTION
**Pull request by Aggermor**
1. Changed some code slightly to think of scaling. Ex. adding more monsters.
2. Fixed some formatting and incorrect calculations

### CODE REVIEW

File: Battleground.cpp
Comment (Aggermor) [inline Battleground.cpp:110]: added error code.  Do you mind it?

File: Battleground.cpp
Comment (Aggermor) [inline Battleground.cpp:202]: we can remove this comment

File: Battleground.cpp
Comment (Aggermor) [inline Battleground.cpp:278]: We could delete this code and use the code below

Comment (Aggermor) [reply Battleground.cpp:278]: Discord thread: [https://discord.com/channels/1122084538726752330/1180797100938760293](https://discord.com/channels/1122084538726752330/1180797100938760293)

**Review by Aggermor — COMMENTED** [review]
general feedback without explicit approval

**Comment by Aggermor** [comment]
I forgot to fetch the original repo before making this pull request. I don't have write access to your repo so I can't resolve conflicts before merging.

also I see changes to `Opponent::randomChoice()` but they don't show up as a changed file here so I'm not sure how this will merge

File: Battleground.cpp
Comment (adrianggggg) [reply Battleground.cpp:110]: not at all

**Review by adrianggggg — APPROVED** [review]

**Review by adrianggggg — APPROVED** [review]

**Pull request merged**
